test/index.test.js

```diff
@@ -81,10 +81,45 @@ describe('DID Identity Provider', function () {
     })
 
     it('DID identity with incorrect id does not verify', async () => {
-      const identity2 = { id: 'NotAnId', publicKey: identity.publicKey, signatures: identity.signatures, type: identity.type, sign: identities.sign, verify: identities.verify }
+      const identity2 = { 
+        ...identity,
+        id: 'NotAnId', 
```

Comment:
i found that the `identities.verifyIdentity` function exited by the "isIdentity" check because the original identity was not complete (had missing fields such as hash, bytes, etc.), I assume that was not the intended behaviour of this test so I added this fix too